Probleme beim Kompilieren

Wenn man das Projekt direkt Kompilieren will erhält man schnell den folgenden Fehler 

`Compiling .pio\build\esp32\lib596\GxEPD2\epd3c\GxEPD2_583c.cpp.o
In file included from .pio\libdeps\esp32\Adafruit GFX Library\Adafruit_GrayOLED.cpp:20:0:
.pio\libdeps\esp32\Adafruit GFX Library\Adafruit_GrayOLED.h:30:32: fatal error: Adafruit_I2CDevice.h: No such file or directory`

Hier fehlt noch die Referenz auf 

`#include <Adafruit_I2CDevice.h>`

--- a/src/hiveeyes-epaper.ino
+++ b/src/hiveeyes-epaper.ino
@@ -32,6 +32,7 @@
 #include "forecast_record.h"
 #include "hive_record.h"
 #include "apicast_record.h"
+#include <Adafruit_I2CDevice.h>
 
 //#include "lang.h"
 //#include "lang_cz.h"                // Localisation (Czech)

